src/components/JaegerIntegration/JaegerHelper.ts

```diff
@@ -40,3 +40,30 @@ export const getTimeRangeMicros = () => {
     to: boundsMillis.to ? boundsMillis.to * 1000 : undefined
   };
 };
+
+const workloadFromNodeRegex = new RegExp(`([a-z0-9-\\.]+)-[a-z0-9]+-[a-z0-9]+\\.([a-z0-9-]+)`);
```

Comment:
Unchanging regexp is better as regexp literal:

`const workloadFromNodeRegex = /([a-z0-9-\\.]+)-[a-z0-9]+-[a-z0-9]+\\.([a-z0-9-]+)/;`